gradio/networking.py

```diff
@@ -117,6 +117,16 @@ def set_share_url_in_config_file(temp_dir, share_url):
     )
 
 
+def set_sample_data_in_config_file(temp_dir, sample_inputs):
```

Comment:
This looks right, but did you actually test it and see if the config.json file looks right? Ideally you would write a test for this function.